config/cl.yml

```diff
@@ -98,6 +98,9 @@ entries:
   - from: /releases/2023-01-09/cl.obo
     to: https://github.com/obophenotype/cell-ontology/releases/download/v2023-01-09/cl.obo
 
+- prefix: /images/
+  replacement: https://github.com/obophenotype/cell-ontology/tree/master/images/
```

Comment:
I think this has to be a 'raw' URL or you'll get HTML instead of an image:

`replacement: https://raw.githubusercontent.com/obophenotype/cell-ontology/master/images/`

E.g. https://raw.githubusercontent.com/obophenotype/cell-ontology/master/docs/CL-logo.jpeg